### PR TITLE
INTMDB-460 third party integrations

### PR DESCRIPTION
**Pull request by ParthasarathyVaradhan**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s): 

## Type of change:

- [ ✅ } Bug fix (non-breaking change which fixes an issue)
- [ ✅  ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ✅  ] This change requires a documentation update

## Required Checklist:

- [ ✅ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ✅  ] I have added tests that prove my fix is effective or that my feature works
- [ ✅  ] I have added any necessary documentation (if appropriate)
- [✅  ] I have run `make fmt` and formatted my code

## Further comments

Examples for each integration is present in examples folder.

### CODE REVIEW

**Review by matt-condon — APPROVED** [review]
LGTM, a couple of small clarifying questions

File: cfn-resources/third-party-integration/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+
+_Type_: String
+
+_Allowed Values_: <code>PAGER_DUTY</code> | <code>MICROSOFT_TEAMS</code> | <code>SLACK</code> | <code>DATADOG</code> | <code>NEW_RELIC</code> | <code>OPS_GENIE</code> | <code>VICTOR_OPS</code> | <code>FLOWDOCK</code> | <code>WEBHOOK</code> | <code>PROMETHEUS</code>
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+
+#### LicenseKey
+
+Unique 40-hexadecimal digit string that identifies your New Relic license.
```
[the remark this answers is not in the document]
Comment (ParthasarathyVaradhan) [reply cfn-resources/third-party-integration/docs/README.md]: Thats a good catch, NEW_RELIC has reached EOL, I have removed NEW_RELIC related variables in https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/146/commits/b0d3652b23e6ad6aa816cad392cfd47d2496d686

File: cfn-resources/third-party-integration/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+
+#### WriteToken
+
+Insert key associated with your New Relic account.
+
+_Required_: No
+
+_Type_: String
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
```
[the remark this answers is not in the document]
Comment (ParthasarathyVaradhan) [reply cfn-resources/third-party-integration/docs/README.md]: Any files under /docs/ folder is autogenerated by AWS Cloudformation CLI (we can't modify autogenerated docs). These provides info about various properties that involved the current resource.

As far as the above question, the link lets us know the property can be updated without causing any interruption to the actual resource. 
There are 3 types of actions that can happen during an update of a field, ( https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html )
1. Update with No Interruption
2. Updates with Some Interruption
3. Replacement

More info on the resource can be seen in, 
https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/b0d3652b23e6ad6aa816cad392cfd47d2496d686/cfn-resources/third-party-integration/README.md

File: cfn-resources/third-party-integration/docs/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ParthasarathyVaradhan) [reply cfn-resources/third-party-integration/docs/README.md]: More info on Integrations whose status is EOL/BETA is available in https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/b0d3652b23e6ad6aa816cad392cfd47d2496d686/cfn-resources/third-party-integration/README.md

**Comment by Zuhairahmed** [comment]
This branch has conflicts, can you resolve? if not ping me tomorrow and we can do together 

image

**Review by Zuhairahmed — COMMENTED** [review]
Few more comments below:

File: cfn-resources/third-party-integration/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| FLOWDOCK | EOL                                                | [From MongoDB](https://www.mongodb.com/docs/atlas/tutorial/third-party-service-integrations/?_ga=2.141767858.1639178218.1667927805-1433452924.1667927805)                                                                                                       |
+| VICTOR_OPS                | ![Build](https://img.shields.io/badge/Beta-yellow) | [Jira Ticket](https://jira.mongodb.org/browse/HELP-39527)                                                                                                                                                                                                       |
+
+
+For more information see: MongoDB Atlas API ThirdPartyIntegrations [Endpoint](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Third-Party-Service-Integrations) Documentation.
```
Comment (Zuhairahmed) [inline cfn-resources/third-party-integration/README.md:65]:
```suggestion
For more information see: MongoDB Atlas API [ThirdPartyIntegrations Endpoint](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Third-Party-Service-Integrations) Documentation.
```

File: cfn-resources/third-party-integration/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+| NEW_RELIC              | EOL                                                | [From MongoDB](https://www.mongodb.com/docs/atlas/tutorial/third-party-service-integrations/?_ga=2.141767858.1639178218.1667927805-1433452924.1667927805), [From NewRelic](https://discuss.newrelic.com/t/new-relic-plugin-eol-wednesday-june-16th-2021/127267) |
+| FLOWDOCK | EOL                                                | [From MongoDB](https://www.mongodb.com/docs/atlas/tutorial/third-party-service-integrations/?_ga=2.141767858.1639178218.1667927805-1433452924.1667927805)                                                                                                       |
+| VICTOR_OPS                | ![Build](https://img.shields.io/badge/Beta-yellow) | [Jira Ticket](https://jira.mongodb.org/browse/HELP-39527)                                                                                                                                                                                                       |
+
```
[the remark this answers is not in the document]
Comment (ParthasarathyVaradhan) [reply cfn-resources/third-party-integration/README.md:68]: Addressed these in https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/146/commits/6184106086c5bd4b31bd835a1efd8da7126b6391

**Comment by ParthasarathyVaradhan** [comment]
> This branch has conflicts, can you resolve? if not ping me tomorrow and we can do together
> 
> image

Addressed. @Zuhairahmed 
Screenshot 2022-12-02 at 3 29 58 PM

File: cfn-resources/third-party-integration/README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+| NEW_RELIC              | EOL                                                | [From MongoDB](https://www.mongodb.com/docs/atlas/tutorial/third-party-service-integrations/?_ga=2.141767858.1639178218.1667927805-1433452924.1667927805), [From NewRelic](https://discuss.newrelic.com/t/new-relic-plugin-eol-wednesday-june-16th-2021/127267) |
+| FLOWDOCK | EOL                                                | [From MongoDB](https://www.mongodb.com/docs/atlas/tutorial/third-party-service-integrations/?_ga=2.141767858.1639178218.1667927805-1433452924.1667927805)                                                                                                       |
+| VICTOR_OPS                | ![Build](https://img.shields.io/badge/Beta-yellow) | [Jira Ticket](https://jira.mongodb.org/browse/HELP-39527)                                                                                                                                                                                                       |
```
[the remark this answers is not in the document]
Comment (ParthasarathyVaradhan) [reply cfn-resources/third-party-integration/README.md]: Addressed in later commits @Zuhairahmed

**Review by Zuhairahmed — APPROVED** [review]
Thanks for updates @ParthasarathyVaradhan. LGTM!

**Review by govardhanpagidi — APPROVED** [review]
LGTM!

**Pull request merged**